static/templates/settings/display-settings.handlebars

```diff
@@ -48,7 +48,7 @@
                     {{/if}} />
                     <span></span>
                 </label>
-                <label for="emoji_alt_code" class="inline-block">{{t 'Display emoji reactions as text (:heart: instead of <img alt=":heart:" class="emoji" src="/static/generated/emoji/images/emoji/unicode/2764.png" title=":heart:"/>).' }}</label>
+                <label for="emoji_alt_code" class="inline-block">{{t 'Emoji reactions as text (:heart: instead of <img alt=":heart:" class="emoji" src="/static/generated/emoji/images/emoji/unicode/2764.png" title=":heart:"/>)' }}</label>
```

Comment:
I think this description is less clear than the old one.  Though we may soon end up folding it into an "emoji style" setting that has this + Google/Twitter/Etc.